h/resources.py

```diff
@@ -106,6 +108,32 @@ def __getitem__(self, client_id):
             raise KeyError()
 
 
+class OrganizationFactory(object):
+    def __init__(self, request):
+        self.request = request
+
+    def __getitem__(self, pubid):
+        try:
+            return self.request.db.query(Organization).filter_by(pubid=pubid).one()
+        except exc.NoResultFound:
+            raise KeyError()
```

Comment:
This `KeyError` causes h to 404 if no group with the given `pubid` (from the URL) exists

Reply:
Would you be so kind as to add a comment to this effect? Could be v. helpful later on!